### PR TITLE
Cleaned up data generation script for ease-of-use and added cards.pck

### PR DESCRIPTION
**Pull request by jacobstorms**
Removed "generate_data.py" and replaced it with "generate-data.py". Added a bunch of whitespacing where necessary and improved readability overall in generate-data. Added a previously generated pickle file of the card images to tools/data so that the script doesn't have to generate it every time and so I could cut that part of the code out. Added instructions on how to download the describable textures dataset how to edit the script s.t. a correctly located .pck file is generated. Tyler and I are going to test this tonight on his machine to make sure everything works and we can move on from this. I'll pull this onto my PC (because I have plenty of space) and start generating training data there overnight

### CODE REVIEW

**Review by TylerLawson — APPROVED** [review]
Had to install correct libraries with pip, but everything else ran as expected. Nice job

File: Tools/generate-data.py [hunk 1]
```diff
@@ -0,0 +1,433 @@
+"""
+This script draws heavily from the one at
+
+https://github.com/geaxgx/playing-card-detection/blob/master/creating_playing_cards_dataset.ipynb
```
Comment (CollinDietz) [inline Tools/generate-data.py:4]: Good job having this citation

**Comment by jacobstorms** [comment]
"pip3 install opencv-python" should work to install openCV on macOS for Python3

**Comment by CollinDietz** [comment]
LGTM

**Review by CollinDietz — APPROVED** [review]

**Pull request merged**